src/app/(evalify)/(faculty)/question-bank/[bankId]/page.tsx

```
@@ -140,10 +193,92 @@ export default function QuestionBankPage() {
+        }
+
+        // Apply sorting if selected
+        if (sortBy && questions) {
+            questions = [...questions].sort((a, b) => {
+                let aValue: string | number;
+                let bValue: string | number;
+
+                switch (sortBy) {
+                    case "difficulty":
+                        // Define difficulty order: EASY < MEDIUM < HARD
+                        const difficultyOrder = { EASY: 1, MEDIUM: 2, HARD: 3 };
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)